modules/nested-tabs/assets/js/editor/module.js

```diff
@@ -5,4 +5,3 @@ export default class Module {
 		elementor.elementsManager.registerElementType( new NestedTabs() );
 	}
 }
-
```

Comment:
?

Reply:
Done

Reply:
I don't understand why we still see this change.